docs/use_case/postv1.md

```diff
@@ -0,0 +1,61 @@
+This initial version of the demo #post app will allow you to create and visualise your own post(s). It will also showcase examples of the ONEM's python SDK usage.
+
```

Comment:
I would like to see a description of the project layout and the different files and their purpose.

Reply:
All of the comments on this PR have been resolved, except for this one. So, should I briefly explain to the user/dev the layout of this Django project ? Eg. what's the  role of the `views.py`, `urls.py` files etc, how is the project pieced together etc?
If so, I can do this in the Introduction part as it will apply to all 3 of the sub-menus: post v1, post v2, post v3.

Reply:
Yes ok by me

Reply:
@chrishornmem - already done that. Please check commit:
https://github.com/ONEmCommunications/documents/pull/15/commits/e435560b106017e3c768d4b6669bc5673b47c495

and let me know if it looks ok. I tried to keep it short but concise.